### PR TITLE
feat: remove MultiAction and TouchAction

### PR DESCRIPTION
**Pull request by KazuCocoa**
We start removing MultiAction and TouchAction endpoints on the server side so clients can also start removing them.
In Python client, we already marked these commands as deprecated in v2, which is 2 years+ ago

Let's see how CI will be..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ For example, some changes in the Selenium binding could break the Appium client.
 > to keep compatible version combinations.
 
 
+### Quick migration guide from v3 to v4
+- Removal
+    - `MultiAction` and `TouchAction` are removed
```
[the remark this answers is not in the document]
Comment (KazuCocoa) [reply README.md:64]: Yea, after ci check completes, will add description to `Quick migration guide from v1 to v2`, or to `MultiAction/TouchAction to W3C actions` and a few modification in both sections to refer to the same place

**Comment by KazuCocoa** [comment]
ok, updated. If looks good, I'll merge this and create a new major version cc @mykola-mokhnach

File: README.md [hunk 2]
```diff
@@ -59,6 +59,16 @@ For example, some changes in the Selenium binding could break the Appium client.
 > to keep compatible version combinations.
 
 
+### Quick migration guide from v3 to v4
+- Removal
+    - `MultiAction` and `TouchAction` are removed. Please use W3C WebDriver actions or `mobile:` extensions
+        - e.g.
```
Comment (mykola-mokhnach) [inline README.md:65]: this paragraph is not needed

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by mykola-mokhnach** [comment]
Do not forget to add BREAKING CHANGE: comments into the PR

**Pull request merged**